tc/sgx/trusted_worker_manager/enclave/wpe/CMakeLists.txt

```diff
@@ -83,3 +83,10 @@ LINK_ENCLAVE_COMMON_LIBRARIES()
 INCLUDE_ENCLAVE_COMMON_DIRECTORIES()
 INCLUDE_ENCLAVE_COMMON_TARGET_DIRECTORIES()
 TARGET_INCLUDE_DIRECTORIES(${PROJECT_NAME} PRIVATE ${TCF_TOP_DIR}/examples/apps)
+
+SET(MRENCLAVE_TARGET GenerateMREnclave)
+ADD_CUSTOM_TARGET(${MRENCLAVE_TARGET} ALL COMMAND
+        COMMAND ${TCF_TOP_DIR}/scripts/generate_mrenclave.sh ${SIGNED_ENCLAVE} ${TCF_TOP_DIR}/wpe_mr_enclave.txt
+        )
```

Comment:
Can we use `add_custom_command` with POST_BUILD

Reply:
Done